packages/components/carousel/src/carousel.vue

```diff
@@ -282,6 +294,10 @@ function resetTimer() {
   startTimer()
 }
 
+function setContainerHeight(height: number) {
+  containerHeight.value = height
```

Comment:
```suggestion
if (props.height === 'auto') {
  containerHeight.value = height
}
```

When height is not `auto`, extra computed should be avoided

Reply:
ok , i see